### PR TITLE
www-apps/grafana-bin: bump version to 5.2.3, remove old

### PR DESCRIPTION
**Pull request by ferki**
Please review, then merge or let me know how
to improve :)

Closes: https://bugs.gentoo.org/664830
Package-Manager: Portage-2.3.48, Repoman-2.3.10

### CODE REVIEW

File: www-apps/grafana-bin/Manifest [hunk 1]
```diff
@@ -1,2 +1 @@
-DIST grafana-bin-5.1.3.tar.gz 52436314 BLAKE2B e5ff613a6d65018b6c205a88a520b5f07ea32955cb8f4b75d1eef05946d93c50272edecf80c8a4a6b8b4a70fbae7c90c71f1b833a31ff56ed71c59dda3007b4c SHA512 3d0fcc2015d95a5334d571f4ea61fa1bb7ead1b4131342e3bf167c832c0f951ae6669049da73a8d954117d54b5422be9203505db19b370ec337c481900d7fea5
```
Comment (Zlogene) [inline www-apps/grafana-bin/Manifest]: make your commits atomic first, one for removal and one for addition

Comment (ferki) [reply www-apps/grafana-bin/Manifest]: Alright, thanks for the feedback! First I did it like that, but then I figured one change would not make sense without the other and squashed them. Will split and push again in a bit.

Comment (ferki) [reply www-apps/grafana-bin/Manifest]: @Zlogene I reverted my squash.

File: www-apps/grafana-bin/grafana-bin-5.2.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user systemd
+
+MY_PN=${PN/-bin/}
+S=${WORKDIR}/${MY_PN}-${PV}
```
Comment (Zlogene) [inline www-apps/grafana-bin/grafana-bin-5.2.3.ebuild]: moved brfore `QA_EXECSTACK` so the order looks just better

File: www-apps/grafana-bin/grafana-bin-5.2.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+QA_EXECSTACK="usr/share/grafana/vendor/phantomjs/phantomjs"
+QA_PRESTRIPPED=${QA_EXECSTACK}
+
+pkg_setup() {
+	enewgroup grafana
+	enewuser grafana -1 -1 /usr/share/grafana grafana
+}
+
+src_install() {
+	keepdir /etc/grafana
```
Comment (Zlogene) [inline www-apps/grafana-bin/grafana-bin-5.2.3.ebuild]: do you really need to keepdir the directory before installing into it? it wil not be empty in anyway.

**Comment by ferki** [comment]
@Zlogene, thanks for the review! I really hoped for a quick merge due to the security fix included in 5.2.3, and since it was just a file rename, but I'm happy to address your other change requests too. Unfortunately I already force pushed the split commits to my original branch before you made that part of the review, so they are marked as outdated now :/

Would you like me to make the bump on one commit and your requested changes on another, or can they go into the same commit?

Or perhaps should we make the bump and the cleanup of the old on this PR, and then I open another PR with your requested updates (and would you need a revbump then)?

**Comment by ferki** [comment]
Nevermind, looks like Patrick already bumped it in the tree. I already updated the PR though.

**Pull request closed without merging**